go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/lightninglabs/aperture v0.1.3-beta
 	github.com/lightninglabs/faraday v0.2.2-alpha
 	github.com/lightninglabs/lndclient v0.11.0-3
-	github.com/lightninglabs/loop v0.11.1-beta
+	github.com/lightninglabs/loop v0.11.2-beta
```

Comment:
Crap! I though we were on `v0.11.2` of `loop` already.

Reply:
No problem, gives us a good reason to do a minor release of LiT which then also will trigger the ARM docker image builds for the first time.